styles.css

```diff
@@ -472,6 +476,79 @@ h1::after {
   margin-top: 10px;
 }
 
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  padding-bottom: 2rem;
+  margin-top: 4578px;
+}
+
+.contact-option-lists {
+  width: 327px;
+  height: 216px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 54px;
+  margin-top: 73px;
+}
+
+.contact-form li {
+  list-style: none;
+}
+
+input {
+  width: 295px;
+  height: 18px;
+  padding-top: 1.5rem;
+  padding-bottom: 1.5rem;
+  resize: none;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  margin-left: -14px;
+  margin-right: -14px;
+}
+
+textarea {
+  height: 114px;
+  width: 295px;
+  border-bottom: 1px solid #dbd8d7;
```

Comment:
- [ ] Please remove any border from the textarea tag. As you can see in the design there is no border in all sides of the textarea.